kubecost-masterpayer-account-permissions.yaml

```
@@ -4,7 +4,7 @@ AWSTemplateFormatVersion: '2010-09-09'
 Description: Kubecost Masterpayer Account Template
 
 Parameters:
-  AthenaCURBucket:
+  S3CURBucket:
     Description: The bucket where the CUR is sent from the “Setting up the CUR” step.
```

Nit. Do we want to update this to the same as the other CFT?
```suggestion
    Description: The bucket created to receive CUR data upon CUR creation.
```